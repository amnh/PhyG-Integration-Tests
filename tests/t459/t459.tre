(AlbanianTosk:73.55,((Manx:168.70000000000002,OldPrussian:52.95)HTU19:58.6,(Irish:206.1,(((Faroese:46.1,Scots:79.6)HTU21:33.3,((Yiddish:115.80000000000001,((Assamese:52.800000000000004,SanskritVedic:32.6)HTU24:22.1,(AncientIonicGreek:43.8,Danish:55.55)HTU26:19.6)HTU25:20.1)HTU18:11.55,(EasternFrisianSaterland:101.95,(Icelandic:92.1,KolschColognian:26.4)HTU20:30.6)HTU30:21.1)HTU22:43.3)HTU27:69.6,(ArmenianWestern:45.95,Cornish:183.1)HTU29:54.1)HTU23:7.95)HTU28:47.2)HTU17:60.05)HTU16[1104.7000000000003];
