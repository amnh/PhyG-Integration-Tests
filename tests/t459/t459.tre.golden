(Afrikaans:141.2,(Welsh:140.65,(Manx:144.25,((AlbanianTosk:44.65,TocharianA:38.550000000000004)HTU57:60.65,((ClassicalArmenian:159.54999999999998,(Latgalian:206.9,((Cornish:172.95000000000002,(ArmenianWestern:105.55,(Lithuanian:202.4,(Latvian:170.15,(EnglishBritish:35.25,(Faroese:68.65,(Scots:76.15,(WesternFrisian:43.65,((Danish:42.65,((SouthernTsakonian:46.55,((PharasaGreek:110.65,(GreekModern:43.65,(Norweigan:104.45,Swedish:96.25)HTU75:23.75)HTU94:8.35)HTU76:38.65,(EnglishAmerican:31.65,(CentralFranconianRipuarian:168.65,(Gothic:158.54999999999998,(AncientGreek:164.35,Yiddish:90.65)HTU65:8.95)HTU64:18.15)HTU68:33.15)HTU89:48.95)HTU90:32.85)HTU74:19.15,(AncientIonicGreek:44.85,(SanskritVedic:35.15,(Avestan:23.55,(Assamese:20.75,Luwian:122.95)HTU71:18.2)HTU69:29.35)HTU70:35.15)HTU93:20.25)HTU61:19.05)HTU58:11.5,(EasternFrisianSaterland:107.15,(NorthFrisianOomrang:130.15,(PennsylvaniaDutch:61.5,(German:32.15,((LowSaxon:43.05,(OldHighGerman:33.65,(Dutch:158.25,(Luxemburgish:60.15,SchwyzerdutschSwissGerman:63.35)HTU87:9.25)HTU73:20.15)HTU72:48.8)HTU62:8.15,(Icelandic:86.35,KolschColognian:35.75)HTU67:16.65)HTU95:27.65)HTU98:34.35)HTU60:9.549999999999999)HTU66:17.65)HTU63:9.15)HTU92:24.25)HTU78:32.85)HTU77:21.65)HTU96:19.25)HTU97:36.15)HTU54:7.15)HTU80:54.65)HTU53:30.65)HTU86:39.15,((OldPrussian:48.15,(Breton:178.7,Samogitian:168.75)HTU51:8.75)HTU79:52.25,(ArmenianEastern:29.05,TocharianB:80.0)HTU81:43.85)HTU91:19.2)HTU82:18.15)HTU56:7.35)HTU55:14.65,(Irish:205.65,(OldIrish:164.55,ScottishGaelic:163.05)HTU85:8.75)HTU84:24.65)HTU88:32.199999999999996)HTU83:13.65)HTU59:52.95)HTU52:8.2)HTU50[2496.2];
